bundle.Dockerfile

```diff
@@ -15,7 +15,7 @@ LABEL com.redhat.delivery.operator.bundle=true
 
 # This second label tells the pipeline which versions of OpenShift the operator supports.
 # This is used to control which index images should include this operator.
-LABEL com.redhat.openshift.versions="=v4.17"
+LABEL com.redhat.openshift.versions="=v4.16"
```

Comment:
is this also needed in annotations.yaml?

https://github.com/openshift/windows-machine-config-operator/blob/master/bundle/metadata/annotations.yaml#L12

Reply:
Not sure, I will add it just in case